core/src/core-api/_internalLinkManager.js

```diff
@@ -125,6 +136,10 @@ export class linkManager extends LuigiCoreAPIBase {
     });
   }
 
+  getCurrentRoute() {
```

Comment:
would be nice to just have a small JSDoc on this fuction only. More specifically to explain how we get to the **buildPathForGetCurrentRoute** function through _app -> ctx -> props -> . Interesting syntax but cumbersome for someone reading the line for the first time.